composer.json

```diff
@@ -21,7 +21,7 @@
     }
   },
   "require": {
-    "php": "~5.5|^7.0",
+    "php": ">=5.5",
```

Comment:
Although it __works__ out of the box, as PHP doesn't follow semantic versioning we cannot rely on it.